### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by Kwentsir**
In this PR, the following were implemented:

> Add a basic redux structure using the duck's pattern

> Install dependencies

> Install linters

### CODE REVIEW

**Comment by RWUBAKWANAYO** [comment]
Hello,
I'm starting to review @Kwentsir  source code

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>,
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (RWUBAKWANAYO) [inline src/index.js:23]: I think it should be better if we remove `reportwebvitals()` function as long as we don't need it.
If you decide to delete it, you should also delete `reportWebVitals.js` file

**Review by RWUBAKWANAYO — APPROVED** [review]
Hi, @Kwentsir,

This part of the project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make our code better. Some of them were simply missed by the previous reviewer and addressing them will really improve our application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/index.js (hunk 1 above)
Comment (Kwentsir) [reply src/index.js:23]: @RWUBAKWANAYO noted, thank you.

**Pull request merged**